(Thu Dec  5 17:50:58 2019)(-0.139065 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/zac/anaconda3/envs/Python37/lib/python3.7/site-packages/pyscannerbit/ScannerBit/lib/libscanner_diver_1.0.0.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-fb0naruj/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(-0.139036 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/zac/anaconda3/envs/Python37/lib/python3.7/site-packages/pyscannerbit/ScannerBit/lib/libscanner_diver_1.0.2.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-fb0naruj/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(-0.127855 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/zac/anaconda3/envs/Python37/lib/python3.7/site-packages/pyscannerbit/ScannerBit/lib/libscanner_great.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-fb0naruj/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(-0.114078 [s])[Default,Scanner][Warning][Non-fatal]:
WARNING: A problem has been raised by ScannerBit.
Could not find plugin library "/home/zac/anaconda3/envs/Python37/lib/python3.7/site-packages/pyscannerbit/ScannerBit/lib/libscanner_multinest_3.10.so".

Raised at: line 102 in function Gambit::Scanner::Plugins::Plugin_Loader::Plugin_Loader() of /tmp/pip-req-build-fb0naruj/pyscannerbit/scannerbit/untar/ScannerBit/ScannerBit/src/plugin_loader.cpp.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0170251 [s])[Default,Utilities]:
Random number engine mt19937_64 (default) selected with seed 3777805.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0175578 [s])[Default,Printers][Info]:
Running shell command: rm -f pyscannerbit_run_data/twalk_scan/samples//results.hdf5
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0232728 [s])[Default,Core]:
Starting scan.
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0257175 [s])[Default,Printers][Info]:
Assigned printer ID '0' to output quantity with label 'LogLike'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0257751 [s])[Default,Printers][Info]:
Assigned printer ID '1' to output quantity with label 'pointID'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0258154 [s])[Default,Printers][Info]:
Assigned printer ID '2' to output quantity with label 'MPIrank'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0267702 [s])[Default,Printers][Info]:
Assigned printer ID '3' to output quantity with label 'mult'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0268121 [s])[Default,Printers][Info]:
Assigned printer ID '4' to output quantity with label 'chain'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.027581 [s])[Default,Printers][Info]:
Assigned printer ID '5' to output quantity with label 'default::z'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0276726 [s])[Default,Printers][Info]:
Assigned printer ID '6' to output quantity with label 'default::x'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0277788 [s])[Default,Printers][Info]:
Assigned printer ID '7' to output quantity with label 'default::y'
--<>--<>--<>--<>--<>--<>--<>--
(Thu Dec  5 17:50:58 2019)(0.0281661 [s])[Default,Printers][Info]:
Assigned printer ID '8' to output quantity with label 'unitCubeParameters'
--<>--<>--<>--<>--<>--<>--<>--
